gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/internal/osUtils.kt

```diff
@@ -68,6 +68,19 @@ internal object MacUtils {
     val xcrun: File by lazy {
         File("/usr/bin/xcrun").checkExistingFile()
     }
+
+    val make: File by lazy {
+        File("/usr/bin/make").checkExistingFile()
+    }
+}
+
+internal object UnixUtils {
+    val sh: File by lazy {
+        File("/bin/sh").checkExistingFile()
```

Comment:
What will happen if tools not found?

Reply:
Maybe add an informative description of the error?

Reply:
By the way, I removed usage of redundant tools /bin/sh and /usr/bin/xcode-select